etc/kayobe/pulp-host-image-versions.yml

```diff
@@ -1,5 +1,5 @@
 ---
 # Overcloud host image versioning tags
 # These images must be in SMS, since they are used by our AIO CI runners
-stackhpc_rocky_9_overcloud_host_image_version: "yoga-20230929T133006"
-stackhpc_ubuntu_jammy_overcloud_host_image_version: "yoga-20230609T120720"
+stackhpc_rocky_9_overcloud_host_image_version: "zed-20231013T123933"
+stackhpc_ubuntu_jammy_overcloud_host_image_version: "zed-20231013T123933"
```

Comment:
I'm aware these need to change, building antelope host images is on the todo list